float64 x_accel
float64 y_accel
float64 yaw
float delta_t
time stamp
bool is_ready
